packages/jest-core/src/TestScheduler.ts

```diff
@@ -157,20 +157,22 @@ export default class TestScheduler {
       );
     };
 
-    const updateSnapshotState = () => {
-      contexts.forEach(context => {
-        const status = snapshot.cleanup(
-          context.hasteFS,
-          this._globalConfig.updateSnapshot,
-          snapshot.buildSnapshotResolver(context.config),
-          context.config.testPathIgnorePatterns,
-        );
-
-        aggregatedResults.snapshot.filesRemoved += status.filesRemoved;
-        aggregatedResults.snapshot.filesRemovedList = (
-          aggregatedResults.snapshot.filesRemovedList || []
-        ).concat(status.filesRemovedList);
-      });
+    const updateSnapshotState = async () => {
+      await Promise.all(
```

Comment:
should these run sequentially rather than in parallel?

Or potentially, calling `await snapshot.buildSnapshotResolver(context.config),` _first_, then `cleanup` in a sync loop like before? I think that makes sense

Reply:
I mean, none of the tests failed when it was running in parallel... 😅 
But you're the boss - I'm happy to change to be sequential 🙂